SearchRotatedSortedArray.py

```diff
@@ -0,0 +1,51 @@
+
+# Time Complexity :O(Log(n))
+# Space Complexity :O(1)
+# Did this code successfully run on Leetcode : yes
+# Any problem you faced while coding this : no
+
+
+# Your code here along with comments explaining your approach
+def search(nums, target):
+    """
+    :type nums: List[int]
+    :type target: int
+    :rtype: int
+    """
+    start = 0
+    end = len(nums)-1
+    while start <=end:
+        mid = (start+end)//2
```

Comment:
practice using (start + (end -start)/2). it'll help you avoid integer overflow error.